docs/09-Advanced.md

```diff
@@ -159,6 +159,19 @@ var myPieChart = new Chart(ctx, {
 });
 ```
 
+In addition to all tooltip properties, a `dataPoints` property is added containing an `array[CustomTooltipDataPoint]`.
+
+#### CustomTooltipDataPoint
```

Comment:
I think we already have a section on TooltipItems in the docs (since they are exposed to other callbacks). Can we link to that section instead of having it duplicated?